docs/Technical-Documentation/Architecture-Decision-Record/021-sendgrid.md

```diff
@@ -0,0 +1,58 @@
+# 1. Record architecture decisions
```

Comment:
This needs to be updated to 21 and relevant title where we typically match the filename.